packages/config/src/android/Facebook.ts

```diff
@@ -1,18 +1,44 @@
-import { Parser } from 'xml2js';
-
 import { ExpoConfig } from '../Config.types';
+import { assert } from '../Errors';
 import {
   addMetaDataItemToMainApplication,
-  Document,
+  AndroidManifest,
   getMainApplication,
+  ManifestActivity,
+  ManifestApplication,
   removeMetaDataItemFromMainApplication,
 } from './Manifest';
-import { readResourcesXMLAsync, ResourceItemXML } from './Resources';
+import { buildResourceItem, readResourcesXMLAsync, ResourceXML } from './Resources';
 import { getProjectStringsXMLPathAsync, removeStringItem, setStringItem } from './Strings';
 import { writeXMLAsync } from './XML';
 
-const facebookSchemeActivity = (scheme: string) => `
-<activity
+function buildXMLItem({
```

Comment:
i imagine we might want to pull these utilities out somewhere more generic in the future